Added 3.9.1 changes to changelog (#646)
MapsIndoors/docs: src/changelogs/android/v3/CHANGELOG.md

### Security     in case of vulnerabilities.
-->

## [3.9.1] 2021-06-10

### Fixed

- Routing animation no longer flashes when being animated
- The MapsPeople logo now shows when using a FragmentActivity

> - Fixed a certificate issue with lower android api’s making live data and tiles unavailable

Reviewer: ```suggestion
- Fixed a certificate issue with lower Android APIs making Live Data and Tiles unavailable
```

> - Selected locations are now shown even if another location was overlapping it with setLocationHideOnIconOverlapEnabled

Reviewer: ```suggestion
- Selected Locations are now shown, even if another Location was overlapping it with `setLocationHideOnIconOverlapEnabled`
```

- Various bug fixes and stability issues

### Added

> - Added two new domain types with default renderings (co2 and humidity)

Reviewer: ```suggestion
- Added two new Domain Types with default renderings (CO2 and Humidity)
```

> - Made default rendered live data badge text easier readable

Reviewer: ```suggestion
- Made default rendered Live Data badge text more readable
```

> - Added a method to change the text size of marker labels with setMapLabelTextSize(int textSize)

Reviewer: ```suggestion
- Added a method to change the text size of marker labels with `setMapLabelTextSize(int textSize)`
```
